cases/util/sep10.js

```diff
@@ -4,9 +4,12 @@ import { fetch } from "./fetchShim";
 
 export default async function getSep10Token(domain, keyPair, signers) {
   if (!signers) signers = [keyPair];
-  let response = await fetch(domain + "/.well-known/stellar.toml");
+  let response = await fetch(domain + ".well-known/stellar.toml");
   const text = await response.text();
   const toml = TOML.parse(text);
+  if (toml.WEB_AUTH_ENDPOINT[toml.WEB_AUTH_ENDPOINT.length - 1] !== "/") {
```

Comment:
Lets do this check in the getTomlFile() function so we only have to do it in one place, and we won't forget to add it in new tests

Reply:
done!